src/sentry/integrations/slack/message_builder/notifications.py

```diff
@@ -30,8 +30,14 @@ def build_deploy_buttons(notification: ReleaseActivityNotification) -> List[Dict
                 project_url = absolute_uri(
                     f"/organizations/{project.organization.slug}/releases/{release.version}/?project={project.id}&unselectedSeries=Healthy/"
                 )
-                buttons.append({"text": project.slug, "type": "button", "url": project_url})
-
+                buttons.append(
+                    {
+                        "text": project.slug,
+                        "name": project.slug,
```

Comment:
If the problem was from the blank text do we need to add `name`?

Reply:
@mgaeta yeah, it doesn't render w/o `name`. I was using [this](https://api.slack.com/docs/messages/builder?msg=%7B%22attachments%22%3A%5B%7B%22text%22%3A%22%20%22%2C%22color%22%3A%22%232788CE%22%2C%22actions%22%3A%5B%7B%22name%22%3A%22boop%22%2C%22text%22%3A%22boop%22%2C%22type%22%3A%22button%22%2C%22url%22%3A%22site.com%22%7D%2C%7B%22name%22%3A%22internal%22%2C%22text%22%3A%22internal%22%2C%22type%22%3A%22button%22%2C%22url%22%3A%22site.com%22%7D%5D%7D%5D%7D) to mess around with the fields and figure out what was needed and not needed